eng/testing/linker/trimmingTests.targets

```diff
@@ -0,0 +1,97 @@
+<Project>
+  <PropertyGroup>
+    <TrimmingTestDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsBinDir)', 'trimmingTests'))</TrimmingTestDir>
+    <TrimmingTestProjectsDir>$([MSBuild]::NormalizeDirectory('$(TrimmingTestDir)', 'projects'))</TrimmingTestProjectsDir>
+    <TestDotNetPath>$([MSBuild]::NormalizePath('$(DotNetRoot)', 'dotnet'))</TestDotNetPath>
+    <ProjectTemplate>$(MSBuildThisFileDirectory)project.csproj.template</ProjectTemplate>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <TestConsoleAppSourceFiles Include="$(MSBuildProjectDirectory)\*.cs" />
```

Comment:
Nit: don't need `\`

Reply:
I actually tested this, you do need it, ProjectDirectory property doesn't come with a slash in the end. You are thinking of MSBuildThisFileDirectory, which does. (Not sure why there is this inconsistency with well known msbuild properties)

Reply:
Ah, right. Disregard.

Reply:
In case you are interested: https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-reserved-and-well-known-properties?view=vs-2019

> MSBuildProjectDirectory	Reserved	The absolute path of the directory where the project file is located, for example C:\MyCompany\MyProduct.
Do not include the final backslash on this property.